scripts/dts/extract_dts_includes.py

```diff
@@ -42,15 +43,19 @@ class Bindings(yaml.Loader):
     _included = []
 
     @classmethod
-    def bindings(cls, compats, yaml_dirs):
+    def bindings(cls, dts_compats, yaml_dirs):
         # Find all .yaml files in yaml_dirs
         cls._files = []
         for yaml_dir in yaml_dirs:
             for root, dirnames, filenames in os.walk(yaml_dir):
                 for filename in fnmatch.filter(filenames, '*.yaml'):
                     cls._files.append(os.path.join(root, filename))
 
-        yaml_list = {'node': {}, 'bus': {}, 'compat': []}
+        compat_to_yaml = {}
```

Comment:
Is the yaml format really of relevance here? Could be compat_to_binding instead.